ibm/mas_devops/roles/cp4d_service/defaults/main.yml

```diff
@@ -8,7 +8,9 @@
 # of our control -- the only version of CP4D that works with MAS 8.7 is v4.0.7, but we can't install that
 # from the IBM operator catalog at present, much more work is needed in this collection to support the operating
 # model of the Cloud Pak for Data team :/
-cpd_product_version: "{{ lookup('env', 'CPD_PRODUCT_VERSION') }}"
+cpd_product_version: "{{ lookup('env', 'CPD_PRODUCT_VERSION') | default(lookup('env', 'CPD_RELEASE_VERSION'), true) | default('4.0.9', true)  }}"
```

Comment:
I vote for simplifying variables here. Suggest to use cpd_release_version / CPD_RELEASE_VERSION in both cpd and cpd_service roles. We don't want users setting conflicting values here, it does not make sense for what we need in MAS.

@durera any comments here?